### ISSUE TITLE
Paired t-test alignment

### ISSUE THREAD

**Issue by AnaFVicente** [ACCLAB/DABEST-python#48]
A question related to the issue: Paired t-test plot 95% CI shifted #46.
![image](https://user-images.githubusercontent.com/7794227/60654450-51e25000-9e4c-11e9-9e70-b30269a368f7.png)

The curve corresponding to 95% CI distribution should be aligned to the black line corresponding to 95% CI. Most of the curve should be inside the black line, only 5% of the date could be outside. On this image it's not the case at all. More than 5% of the CI distribution falls outside the black line. What am I missing? Thanks in advance.

**Comment by josesho** [comment]
Hi @AnaFVicente ,

I think this is a quirk of both the median difference (and the resulting bootstrap), and this particular set of numbers from the example! 

You can inspect and double check the actual numbers with

```python
two_groups_paired.median_difference.results
```

This will present the upper and lower bounds of the [bias-corrected and accelerated](https://www.jstor.org/stable/2246110 bootstrap(`bca_low` and `bca_high`), as well as the actual boostraps used to calculate the bounds and for plotting (`bootstraps`).

In conclusion, the median difference distribution does look weird, but I think this is not a visual / plotting bug but rather a quirk of the maths.....

**Comment by AnaFVicente** [comment]
Thanks for your reply
